scripts/exrrfs_lbc.sh

```diff
@@ -92,9 +92,10 @@ if ! ls ./lbc*.nc; then
 fi
 
 # add/update chemistry species to lbc.nc
-if [[ "${DO_CHEMISTRY^^}" == "TRUE" ]]; then
-  source "${USHrrfs}"/chem_lbc_update.sh
-fi
+# JLS - this is currently unnecessary - init_atmosphere fills variables correctly
+#if [[ "${DO_CHEMISTRY^^}" == "TRUE" ]]; then
+#  source "${USHrrfs}"/chem_lbc_update.sh
+#fi
```

Comment:
@jordanschnell Great that init_atmosphere can now handle the variables correctly.
In this sense, can we remove lines 95-98 and chem_lbs_update.sh? Thanks!